.github/workflows/binderbot.yaml

```diff
@@ -0,0 +1,78 @@
+name: Binderbot
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - development
+      - master
+    paths:
+      - 'examples/*.ipynb'
+      - 'binder-gallery.yaml'
+      - 'doc/source/_static/icepyx_logo.png'
+  pull_request:
+    branches:
+      - development
+      - master
+    paths:
+      - 'examples/*.ipynb'
+      - 'binder-gallery.yaml'
+      - 'doc/source/_static/icepyx_logo.png'
+
+jobs:
+  build:
+    name: Binderbot Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: parse config
+        id: parse
+        uses: CumulusDS/get-yaml-paths-action@v0.0.3
+        with:
+          file: binder-gallery.yaml
+          gallery_repository: gallery_repository
+          binder_url: binder_url
+          binder_repo: binder_repo
+          binder_ref: binder_ref
```

Comment:
> Oh. I think the problem is that the only record of the `Dockerfile` is on my `enable-gallery` branch. `binder-gallery.yaml` is told to look on the `master` branch for the image information and doesn't find anything, so it gets the error:
> 
> ```
> ModuleNotFoundError: No module named 'xarray'
> ```
> 
> Seems like another thing that won't be properly tested until the merge happens.

Would it be possible to set the `binder_ref` to just point to the branch it is building on? E.g. `development` in this case.

Might be good to share a link to the pangeo-gallery documentation somewhere, so that others will know how this can be configured in the future.

Reply:
Yeah, I can change that for a test to see if it builds correctly.

Reply:
Not sure if it will help, but you can try to run Github Actions locally using https://github.com/nektos/act. Might be easier than just trial and error pushes to this Pull Request.